app/src/main/java/org/wikipedia/main/MainFragment.kt

```diff
@@ -471,6 +472,10 @@ class MainFragment : Fragment(), BackPressedHandler, MenuProvider, FeedFragment.
         (requireActivity() as? BaseActivity)?.launchDonateDialog(campaignId = campaignId)
     }
 
+    override fun yearInReviewClick() {
+        startActivity(YearInReviewActivity.newIntent(requireActivity(), AccountUtil.userName))
```

Comment:
We do not need to pass the username as a parameter since you can use the `AccountUtil.userName` directly in the `YearInReviewActivity`.